plugins/battery/battery.plugin.zsh

```diff
@@ -110,13 +110,19 @@ elif [[ "$OSTYPE" = freebsd* ]]; then
 elif [[ "$OSTYPE" = linux*  ]]; then
 
   function battery_is_charging() {
-    ! acpi 2>/dev/null | command grep -v "rate information unavailable" | command grep -q '^Battery.*Discharging'
+    if (( $+commands[acpitool] )); then
+      ! apcitool 2>/dev/null | command grep -q '^\s+Battery.*Discharging'
```

Comment:
typo: `apcitool`

Reply:
Also you need `grep -E` to use `\s+`, I believe. It's not working for me otherwise.

Reply:
Yep fixed